include/xframe/xaxis.hpp

```diff
@@ -581,6 +583,20 @@ namespace xf
     {
         return lhs.less_than(rhs);
     }
+
+    template <class T>
+    inline auto xaxis_generator(T start, T stop, T step = 1) noexcept
```

Comment:
I think we can find a better name, like `make_axis` or `generate_axis`, or `arange_axis` (to be consistent with xtensor)

Reply:
Just `axis` should be fine, we already have a `coordinate` function for `xcoordinate`, a `variable` function for variables